account_coa_menu/views/menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <menuitem id="menu_account_coa_settings"
+        sequence="50"
+        name="Chart of Accounts"
```

Comment:
I think the label "Chart of Accounts" is misleading, because the children menu entries are for Templates, so it's not for the Chart of Account of the current company, but for the Chart of Accounts of new companies that may be created in the future. I would name it "Templates".

Reply:
Indeed ! Thanks a lot. Done.